Library/Formula/node.rb

```diff
@@ -55,42 +49,54 @@ def install
     system "./configure", *args
     system "make", "install"
 
-    resource("npm").stage libexec/"npm" if build.with? "npm"
-  end
+    if build.with? "npm"
+      resource("npm").stage libexec/"npm_i"
```

Comment:
If you like, I can tee up another PR to tweak the language that went in with this one to fix the issues raised? I went with ` npm_i ` to mean ` npm_install ` but I get that's not wholly clear to anyone who isn't in my mind :wink: